### DISCUSSION TITLE
Footer overflows page

### DISCUSSION THREAD

**Discussion by boosh** [saadeghi/daisyui#2953] [Q&A]
I've copied the footer HTML from the site, but it causes the page to scroll horizontally. It seems to be caused by the padding, which for some reason affects the element width. 

There's also some kind of margin on the other elements on the right. Do I need to use some kind of reset on the html/body tags to set the max-width anything? 

<img width="926" alt="Screenshot 2024-04-02 at 13 31 16" src="https://github.com/saadeghi/daisyui/assets/362148/bd714b4f-29b9-4007-97c7-7e8d7a4e0e3c">

**Comment by boosh** [comment]
```
html, body {
    max-width: 100%;
    overflow-x: hidden;
}
```
That fixes it, but is it normal to have to specify this?

**Reply by saadeghi** [reply]
> That fixes it, but is it normal to have to specify this?

No you shouldn't be needing that.

If there is a problem in the layout please provide a minimal reproduction code on Tailwind Play because a screenshot is not helpful and no one can guess what is the cause of the problem.

**Reply by boosh** [reply]
Sorry I switched frameworks. I don't have the code to hand, but pretty much all I had on the page was a menu and footer.
